winspool/winspool.go

```diff
@@ -154,7 +154,7 @@ func convertPrinterState(wsStatus uint32) *cdd.PrinterStateSection {
 		}
 		state.VendorState.Item = append(state.VendorState.Item, vs)
 	}
-	if wsStatus&PRINTER_STATUS_OFFLINE != 0 {
+	if wsStatus&PRINTER_STATUS_OFFLINE != 0 || wsAttributes&PRINTER_ATTRIBUTE_WORK_OFFLINE != 0 {
```

Comment:
That's really useful info. If you summarize that in a comment in the code there (and a similar comment for the CUPS version), I can merge.